### PR TITLE
2.x dockerfile

### PR DESCRIPTION
**Pull request by cake17**
same as #3416 for 2.x branch

### CODE REVIEW

File: README.mdown [hunk 1]
```diff
@@ -4,87 +4,140 @@ CakePHP Documentation
...
 
-This is the official documentation for the CakePHP project. It is available online in HTML, PDF and EPUB formats at http://book.cakephp.org.
+TThis is the official documentation for the CakePHP project. It is available
+online in HTML, PDF and EPUB formats at http://book.cakephp.org.
 
-Requirements
-------------
+Contributing to the documentation is pretty simple. Please read the
+documentation on contributing to the documentation over on [the
+cookbook](http://book.cakephp.org/3.0/en/contributing/documentation.html) for
```
Comment (antograssiot) [inline README.mdown]: 2.0 ?

Comment (cake17) [inline README.mdown]: ah oui exact !

File: README.mdown [hunk 2]
```diff
@@ -4,87 +4,140 @@ CakePHP Documentation
 [![License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](LICENSE.txt)
 [![Build Status](https://img.shields.io/travis/cakephp/docs/master.svg?style=flat-square)](https://travis-ci.org/cakephp/docs)
 
-This is the official documentation for the CakePHP project. It is available online in HTML, PDF and EPUB formats at http://book.cakephp.org.
+TThis is the official documentation for the CakePHP project. It is available
```
Comment (vanquang9387) [inline README.mdown]: Typo: TThis :smile:

Comment (cake17) [inline README.mdown]: Fixed in 30d2bb8c53aefeeb7c02f915a98678a8ae3c859e :smile:

**Comment by vanquang9387** [comment]
@cake17 Can I use docker at the moment?
Just try
```
$ docker run -i -t -v /var/www/html/cakephp-docs/:/data cakephp/docs make html
```
And the result is
```
Unable to find image 'cakephp/docs:latest' locally
Pulling repository docker.io/cakephp/docs
Error: image cakephp/docs:latest not found
```

**Comment by cake17** [comment]
Yes, you first need to build the image in your computer, see [Readme](https://github.com/cakephp-fr/docs-cakephp/blob/master/README.mdown).

We might create an image for the docs and for cakephp as well on dockerhub, see [discussion here](https://github.com/cakephp/app/issues/293). This would avoid the need to build yourself the images.

**Comment by vanquang9387** [comment]
@cake17 
> Yes, you first need to build the image in your computer, see Readme.

Yes, I just followed instruction in Readme.
```
docker build -t cakephp/docs .
```
If you talking about this, I had already run it (took minutes).

P.S: Sorry, I don't have any exp on Docker, just tried following [instruction](https://github.com/cakephp-fr/docs-cakephp/blob/master/README.mdown#build-the-documentation-with-docker).

**Comment by cake17** [comment]
@vanquang9387 if you run `docker images`, `cakephp/docs`'s image is in the output ?

**Comment by vanquang9387** [comment]
No, it's not there.
```
REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
<none>              <none>              9783ad2c375b        3 hours ago         125.2 MB
hello-world         latest              0a6ba66e537a        7 days ago          960 B
debian              jessie              3d88cbf54477        12 days ago         125.2 MB
```

Do I need to run `docker build -t cakephp/docs .` again?

**Comment by cake17** [comment]
It seems the image has not been builded correctly. Do this:

    # to remove the image
    docker rmi 9783ad2c375b
    # re-run the build command
    docker build -t cakephp/docs .

Maybe we can discuss it there http://irc.lc/freenode/cakephp after. (people receive notifications for each of our discussion.. :-))

**Comment by vanquang9387** [comment]
Oh sorry you guys :flushed:
Didn't know that.

**Comment by cake17** [comment]
No problem :wink:, let us know if this works

**Pull request merged**